conda_package_streaming/transmute.py

```diff
@@ -1,44 +1,26 @@
 """
 Convert .tar.bz2 to .conda
 
-Uses `tempfile.SpooledTemporaryFile` to buffer `pkg-*` `.tar` and `info-*`
-`.tar`, then compress directly into an open `ZipFile` at the end.
+Uses ``tempfile.SpooledTemporaryFile`` to buffer ``pkg-*.tar`` and
+``info-*.tar``, then compress directly into an open `ZipFile` at the end.
 `SpooledTemporaryFile` buffers the first 10MB of the package and its metadata in
 memory, but writes out to disk for larger packages.
-
-Conda packages created this way have `info-*` as the last element in the
-`ZipFile`, instead of the first for `.conda` packages created with pre-2.0
-`conda-package-handling`.
 """
 
 from __future__ import annotations
 
-import json
 import os
-import shutil
 import tarfile
-import tempfile
-import zipfile
 from pathlib import Path
-from typing import Callable
+from typing import Callable, Iterator
 
 import zstandard
 
+from .create import ZSTD_COMPRESS_LEVEL, ZSTD_COMPRESS_THREADS, conda_builder
+
 # streams everything in .tar.bz2 mode
 from .package_streaming import CondaComponent, stream_conda_component
 
-# increase to reduce speed and increase compression (levels above 19 use much
-# more memory)
-ZSTD_COMPRESS_LEVEL = 19
-# increase to reduce compression and increase speed
-ZSTD_COMPRESS_THREADS = 1
-
-CONDA_PACKAGE_FORMAT_VERSION = 2
-
-# Account for growth from "2 GB of /dev/urandom" to not exceed ZIP64_LIMIT after
-# compression
-CONDA_ZIP64_LIMIT = zipfile.ZIP64_LIMIT - (1 << 18) - 1
-
```

Comment:
Just a last thought, could you check if those constants are used in any code in github search? Might be worth keeping them around

Reply:
Search turns up empty. We already import the two ZSTD_ constants here, only removing the package version and esoteric zip64 limit.